deploy/redis/values.yaml

```diff
@@ -24,7 +24,8 @@ logConfigs:
 
 metrics:
   image:
-    registry: infracreate-registry.cn-zhangjiakou.cr.aliyuncs.com
+    # if the value of metrics.image.registry is not specified using `--set`, it will be set to the value of 'image.registry' by default
+    registry: ""
     repository: apecloud/agamotto
```

Comment:
agamotto -> oteld ?

Reply:
In open source KubeBlocks we still use the apecloud/agamotto. And It will be dealt with collectively afterwards.